### DISCUSSION TITLE
Santoker R3 to Artisan

### DISCUSSION THREAD

**Discussion by airwavecoffee** [artisan-roaster-scope/artisan#1703] [Q&A]
I recently bought a used santoker r3
I downloaded the santoker app on iPad, it works but not a huge fan.

I want to connect to artisan via usb. I have downloaded different usb drivers and used my Mac and a pc
cannot figure out how to connect. I downloaded the zip files and extracted them is there anything else that needs to be done? what am I doing wrong? Thank you

Currently using Mac 14.6.1
I can use windows 10 if needed (easier)
[artisan-settings.aset.zip](https://github.com/user-attachments/files/17179924/artisan-settings.aset.zip)

**Comment by Terracotta-6** [comment]
artisan wasn't sure about support for its earlier product, the R3, and asked equipment manufacturers.

SANTOKER Coffee TechHunan Santoker Intelligent Technology Co., Ltd. (alibaba.com)
Address: 40-2, gaobeidian xincun, chaoyang district, Beijinglel: + 86 15116517222Website:www.santoker.comEmail address: hustnan@santoker.comWechat: santoker
Distributor "hailang:SANTOKER(THAILAND)CO.,LTD
Contact: TanPhone number:065-0544493Email:santokerthailand@gmail.comAddress: 11/3 M.6 Saimoon, Sankhumpheng, Chiangmai, Thailand
HENAN WINTOP MACHINERY TECHNOLOGY CO.. LTDContactor: Jenny Tse 谢豫彬MP:18203850626Email:info@wintopmachinery.com

**Comment by MAKOMO** [comment]
> I recently bought a used santoker r3 I downloaded the santoker app on iPad, it works but not a huge fan.
> 
How did you connect your iPad to that machine. Via Wifi or Bluetooth (BLE)?

**Reply by Terracotta-6** [reply]
I contacted the supplier and they said  Santoker R3 does not support artisan.  He's talking about software Santoker Coffee roast   software developed  by amateur.

**Comment by MAKOMO** [comment]
Could you test the just released Artisan v3.1.2? That comes with improved Santoker support and might work with your machine.
